### ISSUE TITLE
aXe 3.0 Isn't Flagging Surrounding Text Color Contrast Issue

### ISSUE THREAD

**Issue by rcarrasquillo19** [dequelabs/axe-core#464]
Deque aXe 3.0 isn’t flagging color contrast issue on surrounding text in Storefront pages.  The detail on the issue is reflected in attached email. 

[Axe Is not Flagging the surrounding text Color contrast issue Email.pdf](https://github.com/dequelabs/axe-core/files/1189427/Axe.Is.not.Flagging.the.surrounding.text.Color.contrast.issue.Email.pdf)

**Comment by WilcoFiers** [comment]
Can you give some details on how you're running the rule? Because it's working alright for me. Are you sure you have the experimental rule turned on?

![screenshot of axe-cli with axe 3.0.0-alpha1 successfully testing link-in-text-block rule](https://user-images.githubusercontent.com/530687/28823757-77b50fc2-76bf-11e7-93bb-40dcdf459cc0.png)

Additionally, for future reference, could you just copy over the issue, instead of posting a PDF. That's a little easier to work with, and that way we don't have to be concerned about security.

**Comment by Olina-Roy** [comment]
We are running the aXe plugin on chrome browser.So for surrounding text it does not flag the issue.But when we manually check the color of link and the surrounding text, it is not passing the required color contrast ratio.
The page URL for your reference:
https://www.ally.com/security/handling-suspicious-messages.html

**Comment by marcysutton** [comment]
That is odd, we'll have to dig in. This appears to be the link called out in the PDF, so we don't have to continue opening it:

```
<li class="suspicious-messages active">
	<a href="/security/handling-suspicious-messages.html" data-track-ui="leftrail" data-track-elem="link" data-track-name="Suspicious Messages" data-track-trigger="internal">Suspicious Messages</a>
</li>
```

The background-color is on the LI, and I don't see anything special about it otherwise.

**Comment by Olina-Roy** [comment]
the issue is on the link abuse@ally.com.
 
Following is the markup:
<p>If an email appears to be from Ally but looks suspicious, forward it to us at <a href="mailto:abuse@ally.com" data-track-elem="link" data-track-name="abuse@ally.com" data-track-trigger="email" data-track-position="1">abuse@ally.com</a> then delete it immediately from your inbox. Do not open email attachments or click links in emails if you do not personally know the sender. <strong>Instead of clicking a link in an email, type the URL directly in your browser</strong> or use favorites/bookmarks to access the website.</p>

<p> tag color is #505050 and <a> tag color is #006899

**Comment by marcysutton** [comment]
Ah! Thanks for clarifying. This is why we need details in the Github issue, not in a PDF. We'll look into it.

**Comment by dylanb** [comment]
@WilcoFiers this is because the aXe extension does not run experimental rules and there is no way to turn them on. This is by design. I think we should close this issue.

**Comment by WilcoFiers** [comment]
Oops! I thought you were talking about axe-core 3.0, not about the 3.0 extension. My bad! Yeah so like @dylanb said, the extensions only run those rules that are enabled by default. The experimental rules (of which link-in-text-block) is one. You'll need to use one of our APIs instead if you want more configuration options. I recommend axe-cli. It's pretty easy to install and use.

For now, I'll consider this ticket resolved.

**Comment by dylanb** [comment]
@rcarrasquillo19 you can also contact us about our WorldSpace Attest extension - which has functionality for supporting custom rule sets and custom settings
